src/js/utils/parser.ts

```diff
@@ -81,21 +81,26 @@ export function serialize(val) {
 }
 
 // Returns the integer value a of css string (e.g. '420px')
-export function parseDimension(dimension) {
-    if (typeof dimension === 'string') {
-        if (dimension === '') {
-            return 0;
-        } else if (dimension.lastIndexOf('%') > -1) {
-            return dimension;
-        }
-        return parseInt(dimension.replace('px', ''), 10);
+export function parseDimension(dimension: string): number | string {
+    if (isValidNumber(dimension)) {
```

Comment:
CSS can have non string dimension values. ie: `margin: 0`. This function isn't actually used in player or commercial either. @robwalch my intuition tells me we shouldn't remove it in case it's being used in custom implementations. Let me know if I'm wrong and we can remove the function.